dev/NavigationView/NavigationView.cpp

```diff
@@ -110,7 +110,6 @@ void NavigationView::UnhookEventsAndClearFields(bool isFromDestructor)
     m_paneSearchButtonClickRevoker.revoke();
     m_paneSearchButton.set(nullptr);
 
-    m_buttonHolderGettingFocusRevoker.revoke();
     m_buttonHolderGrid.set(nullptr);
```

Comment:
>m_buttonHolderGrid [](start = 4, length = 18)

I don't know who and when template part for m_buttonHolderGrid is removed. and it look like dead code now and we can delete it

Reply:
It's actually still used in layout.